ckanext/canada/templates/public/datatables/datatables_view.html

```diff
@@ -25,55 +32,32 @@
   &nbsp;
 {%- endblock -%}
 
-{%- block links -%}
-  {{super()}}
-  {%- block meta -%}
-    {{ super() }}
-    {% block adobe_analytics_meta %}
-      {%- block adobe_analytics_title -%}
-      <meta property="dcterms:title" content="{{ g.site_title }}" />
-      {%- endblock -%}
-      <meta property="dcterms:language" content="{{ h.adobe_analytics_lang() }}" />
-      {%- block adobe_analytics_creator -%}
-      {# remove encode when moving from fanstatic - applied as a fix for OPEN-1566 #}
-      <meta property="dcterms:creator" content="{{ _('Treasury Board of Canada Secretariat').encode('ascii', 'xmlcharrefreplace') | safe }}"/>
-      {%- endblock -%}
-      <meta property="dcterms:service" content="OG-GO" />
-      <meta property="dcterms:accessRights" content="{{ h.adobe_analytics_login_required(h.full_current_url()) }}" />
-    {%- endblock -%}
-  {%- endblock -%}
-
-  {%- block google_analytics -%}
-    {% set ga4_id = h.ga4_id() %}
-    {% if ga4_id %}
-      <!-- Google tag (gtag.js) -->
-      <script async src="https://www.googletagmanager.com/gtag/js?id={{ ga4_id }}"></script>
-      <script>
-        window.dataLayer = window.dataLayer || [];
-        function gtag(){dataLayer.push(arguments);}
-        gtag('js', new Date());
-        gtag('config', '{{ ga4_id }}');
-      </script>
-    {% endif %}
-  {%- endblock -%}
-
-  {% block adobe_analytics %}
-    {% if h.adobe_analytics_js() %}
-      <script src="{{ h.adobe_analytics_js() }}"></script>
-    {% endif %}
-  {%- endblock -%}
-{%- endblock -%}
+{% block adobe_analytics_creator %}
+  {%- set owner_org_title = cr.split_bilingual_field(package.organization.title, h.lang()) -%}
+  {%- set owner_2 = 'N/A' -%}
+  {%- set owner_3 = 'N/A' -%}
+  {%- set owner_4 = 'N/A' -%}
+  {% if 'org_section' in package and h.scheming_language_text(package.org_section) | length > 0 %}
+    {% set org_sections = h.scheming_language_text(package.org_section).split(',') %}
+    {%- set owner_2 = (org_sections[0] or 'N/A') | safe | trim -%}
+    {%- set owner_3 = (org_sections[1] or 'N/A') | safe | trim -%}
+    {%- set owner_4 = (org_sections[2] or 'N/A') | safe | trim -%}
+  {% endif %}
+  {{ cr.adobe_analytics_creator(creator=owner_org_title,
+                                owner_1=owner_org_title, owner_2=owner_2,
+                                owner_3=owner_3, owner_4=owner_4) }}
```

Comment:
`N/A` isn't correct in French, why the `| safe` use here? This would all be simpler as a helper (easier to omit owners).

Reply:
`N/A` is just used in some extra Adobe Analytics thing specific to the Government of Canada from what I understand in the GoC documentation for it. The documentation said to use it if no other owners, but you NEED the 4 owners in the piped string which is kind of annoying.

Will move this to a helper though.

Reply:
@wardi okay there is now a helper that does all of this.